### ISSUE TITLE
Sometimes duplicate command line after Ctrl-R reverse search with fzf-tmux

### ISSUE THREAD

**Issue by Strayer** [romkatv/powerlevel10k#1332]
I have a very weird issue that may be hard to reproduce. I'm using https://github.com/junegunn/fzf with the tmux feature enabled. When I autocomplete a command with Ctrl-R the command SOMETIMES gets duplicated as soon as a keypress is done after fzf inserts the command:

https://user-images.githubusercontent.com/310624/113024062-99fbe700-9186-11eb-9c68-c1ad5eff2d70.mov

This is never happening with pure or starship, so I assume there is some kind of race condition when the command line is inserted by fzf/zsh.

I'd love to get to the cause of this issue but am a bit at a loss on how to proceed.

**Comment by romkatv** [comment]
I'm afraid I don't have the time to debug this. If you figure out what's going on, please post an update.

FWIW, powerlevel10k + fzf works fine  in zsh4humans.

**Comment by Strayer** [comment]
Thanks for responding! I know this issue is horribly vague. I experimented a bit more with this and just can't reproduce it. Even when I have a very simple .zshrc with only fzf and powerlevel10k installed directly via git this still happens. When I tried replicating my setup in a Debian VM the problem doesn't occur (same terminal connected to the VM via SSH).

I did find a workaround though. To make sure this isn't connected to my tmux version I installed the current git master of tmux. It still showed the problem, _but_ when I enable the new popup functionality for fzf with `export FZF_TMUX_OPTS="-p"` it doesn't duplicate the command after completion anymore. Not ideal, but at least it works now. 🤷‍♂️
